### PR TITLE
Update gir-files

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

File: gdk4/src/auto/texture_downloader.rs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            let ret = from_glib_full(ffi::gdk_texture_downloader_download_bytes(
+                self.to_glib_none().0,
+                out_stride.as_mut_ptr(),
+            ));
+            (ret, out_stride.assume_init())
+        }
+    }
+
+    //#[doc(alias = "gdk_texture_downloader_download_into")]
+    //pub fn download_into(&self, data: &[u8], stride: usize) {
```
Comment (sdroege) [inline gdk4/src/auto/texture_downloader.rs:44]: Adding this requires adding checks for the size of the slice and the validity of the stride

File: gtk4/src/auto/scale_button.rs [hunk 2]
```diff
@@ -333,6 +339,8 @@ pub trait ScaleButtonExt: 'static {
     #[doc(alias = "gtk_scale_button_set_value")]
     fn set_value(&self, value: f64);
 
+    fn get_property_active(&self) -> bool;
```
Comment (sdroege) [inline gtk4/src/auto/scale_button.rs:342]: Missing version

File: gtk4/src/auto/accessible.rs [hunk 3]
```diff
@@ -53,6 +106,73 @@ impl<O: IsA<Accessible>> AccessibleExt for O {
         }
     }
 
+    fn at_context(&self) -> ATContext {
```
Comment (sdroege) [inline gtk4/src/auto/accessible.rs:109]: Missing version

File: gsk4/sys/src/lib.rs [hunk 4]
```diff
@@ -1412,6 +1475,24 @@ extern "C" {
     ) -> *mut GskTextureNode;
     pub fn gsk_texture_node_get_texture(node: *const GskTextureNode) -> *mut gdk::GdkTexture;
 
+    //=========================================================================
+    // GskTextureScaleNode
+    //=========================================================================
+    pub fn gsk_texture_scale_node_get_type() -> GType;
```
Comment (sdroege) [inline gsk4/sys/src/lib.rs:1481]: Missing version

File: gtk4/src/auto/scale_button.rs (hunk 2 above)
Comment (sdroege) [reply gtk4/src/auto/scale_button.rs:342]: Fixed already in ad7d9bc409e45968280b101ed38e00d8d104d4cd

File: gtk4/src/auto/accessible.rs (hunk 3 above)
Comment (sdroege) [reply gtk4/src/auto/accessible.rs:109]: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/5537

File: gsk4/sys/src/lib.rs (hunk 4 above)
Comment (sdroege) [reply gsk4/sys/src/lib.rs:1481]: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/5537

File: gtk4/src/auto/scale_button.rs (hunk 2 above)
Comment (sdroege) [reply gtk4/src/auto/scale_button.rs:342]: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/5529

File: gdk4-wayland/Cargo.toml [hunk 5]
```diff
@@ -14,6 +14,7 @@ rust-version = "1.64"
 
 [features]
 v4_4 = ["ffi/v4_4"]
+v4_10 = ["ffi/v4_10"]
```
Comment (bilelmoussaoui) [inline gdk4-wayland/Cargo.toml:17]:
```suggestion
v4_10 = ["ffi/v4_10", "v4_4"]
```
also the new feature should be added to the README file

File: gdk4-x11/src/auto/x11_display.rs [hunk 6]
```diff
@@ -106,6 +107,8 @@ impl X11Display {
         unsafe { from_glib_none(ffi::gdk_x11_display_get_screen(self.to_glib_none().0)) }
     }
 
+    #[cfg_attr(feature = "v4_10", deprecated = "Since 4.10")]
```
Comment (bilelmoussaoui) [inline gdk4-x11/src/auto/x11_display.rs:110]: the crate needs a v4_10 feature

File: gtk4/src/auto/editable.rs [hunk 7]
```diff
@@ -24,6 +27,11 @@ impl Editable {
 }
 
 pub trait EditableExt: 'static {
+    #[cfg(any(feature = "v4_10", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v4_10")))]
+    #[doc(alias = "gtk_editable_delegate_get_accessible_platform_state")]
+    fn delegate_get_accessible_platform_state(&self, state: AccessiblePlatformState) -> bool;
```
Comment (bilelmoussaoui) [inline gtk4/src/auto/editable.rs:33]: this looks like a wrongly annotated API

File: gtk4/src/auto/search_entry.rs [hunk 8]
```diff
@@ -85,12 +109,12 @@ impl SearchEntry {
     }
 
     #[doc(alias = "placeholder-text")]
-    pub fn placeholder_text(&self) -> Option<glib::GString> {
+    pub fn get_property_placeholder_text(&self) -> Option<glib::GString> {
```
Comment (bilelmoussaoui) [inline gtk4/src/auto/search_entry.rs:112]: this getter/setter should no longer be generated

File: gtk4/src/auto/editable.rs (hunk 7 above)
Comment (sdroege) [reply gtk4/src/auto/editable.rs:33]: No, that's actually correct according to the C API.

**Comment by sdroege** [comment]
Also https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/5539

File: gsk4/src/auto/texture_scale_node.rs [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+                texture.as_ref().to_glib_none().0,
+                bounds.to_glib_none().0,
+                filter.into_glib(),
+            ))
+        }
+    }
+
+    #[doc(alias = "gsk_texture_scale_node_get_filter")]
+    #[doc(alias = "get_filter")]
+    pub fn filter(node: impl AsRef<TextureNode>) -> ScalingFilter {
```
Comment (sdroege) [inline gsk4/src/auto/texture_scale_node.rs:44]: Also https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/5539

Comment (sdroege) [reply gsk4/src/auto/texture_scale_node.rs:44]: That unfortunately can't be overridden so let's wait for new .gir files...

File: gdk4-wayland/Cargo.toml (hunk 5 above)
Comment (bilelmoussaoui) [reply gdk4-wayland/Cargo.toml:17]: This wasn't fixed?

File: gtk4/sys/src/lib.rs [hunk 10]
```diff
@@ -1191,13 +1196,49 @@ pub struct _GtkATContextClass {
...
+        Option<unsafe extern "C" fn(*mut GtkAccessible) -> *mut GtkAccessible>,
+    pub get_bounds: Option<
+        unsafe extern "C" fn(
+            *mut GtkAccessible,
+            *mut c_int,
+            *mut c_int,
+            *mut c_int,
+            *mut c_int,
+        ) -> gboolean,
+    >,
```
Comment (bilelmoussaoui) [inline gtk4/sys/src/lib.rs:1220]: Interesting, I will look into adding subclassing support for this before doing the next release.

**Pull request merged**